tests/unit/test_cli.py

```diff
@@ -76,6 +76,30 @@ def test_version_option(self):
         assert lines[0] == f"cachi2 {expect_version}"
         assert lines[1].startswith("Supported package managers: gomod")
 
+    @pytest.mark.parametrize(
+        "file_create, file, file_text, invoke_expectation",
+        [
+            (True, "config.yaml", "gomod_download_max_tries: 1000000", invoke_expecting_sucess),
```

Comment:
Should we test more scenarios here, especially multiple [options](https://github.com/containerbuildsystem/cachi2/blob/main/cachi2/core/config.py#L4) passed via config? Do we want also add some simple check that option was passed and worked correctly?